monkestation/code/__HELPERS/unsorted.dm

```diff
@@ -3,7 +3,7 @@
 	var/list/picked_lockers = list()
 	var/turf/object_location
 	for(var/obj/structure/closet/find_closet in world)
-		if(!istype(find_closet,/obj/structure/closet/secure_closet))
+		if(!find_closet.locked)
```

Comment:
Note that this will increase the number of possible locations. Unlocked head lockers will now be an option.

Reply:
Yeah it's kind of a shift.  Fewer locked boxes, but more unlocked boxes.  I think it's probably fine because it's just more incentive to lock your lockers.